### ISSUE TITLE
Tab navigation on meetings pages, etc is not keyboard focusable

### ISSUE THREAD

**Issue by drnelson6** [AmericanPhilosophicalSociety/video-experiment#63]
This is a violation of accessibility best practices. This should be keyboard focusable to enable navigation with only a keyboard.

**Comment by drnelson6** [comment]
Alphabetic filters are also not keyboard focusable. Should they be?

**Comment by drnelson6** [comment]
Links are only focusable if they have an `href` attribute. This seems to be an inherent accessibility issue in HTMX. Can be fixed by applying `tabindex="0"` to the element. Does this need some ARIA component or other accessibility marker?
